### ISSUE TITLE
Compatible with Binary Classification?

### ISSUE THREAD

**Issue by sumit7692** [asappresearch/aum#10]
Can it be used with Binary classification problem?

**Comment by howtodowtle** [comment]
You call it on the logits, so should work just the same for binary classification.
